package.json

```diff
@@ -91,10 +89,8 @@
     "shiro-trie": "^0.4.8",
     "simple-statistics": "^0.7.0",
     "socket.io": "^2.1.1",
-    "swagger-ui-dist": "^3.20.1",
```

Comment:
swagger-ui-dist is needed for /api-docs.html

Reply:
see https://github.com/nightscout/cgm-remote-monitor/blob/dev/static/api-docs.html and https://github.com/nightscout/cgm-remote-monitor/search?q=swagger-ui-dist&unscoped_q=swagger-ui-dist